src/components/Navigator/QuickNavigationModal.vue

```diff
@@ -311,6 +307,11 @@ export default {
     formatSymbolTitle(symbolTitle, symbolStart, symbolEnd) {
       return symbolTitle.slice(symbolStart, symbolEnd);
     },
+    isFocused(index) {
+      if (this.focusedInput || this.focusedIndex !== index) return false;
+      this.focusReference();
```

Comment:
`isFocused` is used in the template, on the `tabindex`.

I know generally it should be fine to do this, but we are relying on the vue template reactivity to call a mutation method. 

Vue caches or hoists templates in various ways and I dont know what could go wrong here.

Reply:
Not using it anymore https://github.com/apple/swift-docc-render/pull/736/commits/8d8c17e136086ff633c105f2f4aa570e54361551